lib/ui/home/HomeDailyIlliniWidget.dart

```diff
@@ -184,9 +211,11 @@ class _HomeDailyIlliniWidgetState extends State<HomeDailyIlliniWidget> implement
     _pageController?.nextPage(duration: Duration(milliseconds: 500), curve: Curves.easeIn);
   }
 
-  void _onViewAll() {
-    Analytics().logSelect(target: "View All", source: widget.runtimeType.toString());
-    Navigator.push(context, CupertinoPageRoute(builder: (context) => DailyIlliniListPanel()));
+  _onViewAll() async {
+    final Uri url = Uri.parse('https://dailyillini.com');
```

Comment:
This url should be received from the Config service and should not be hardcoded.
Or to leave it with the current implementation - open panel with a list of daily illini items.

Reply:
Done, the url is now fetched from `otherUniversityServices['daily_illini_homepage_url']`. Can it be added as `https://dailyillini.com` on the config servers?

Reply:
Yes, this new entry can be added to the configs once we merge the request.